src/replication.c

```diff
@@ -1308,6 +1308,9 @@ int replicaPutOnline(client *slave) {
                           NULL);
     serverLog(LL_NOTICE,"Synchronization with replica %s succeeded",
         replicationGetSlaveName(slave));
+
+    /* Replicate slot being migrated/imported to the new replica */
+    replicateOpenSlots();
```

Comment:
this will break the CLIENT PAUSE WRITE status, since it doesn't check `server.paused_actions`

Reply:
This is the replica online path so I don't see how clients could be paused for write?